src/app/pages/public/CreateLifePage/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, DatePicker } from 'antd';
+import { useNavigate } from 'react-router-dom';
+import BasicLayout from '../../../layouts/BasicLayout';
+import { useCreateLifeMutation } from '../../../api';
+
+const validateMessages = {
+    required: '${label} is required!',
+};
+
+const layout = {
+    labelCol: {
+        span: 8,
+    },
+    wrapperCol: {
+        span: 16,
+    },
+};
+
+const CreateLifePage = () => {
+    const navigate = useNavigate();
+
+    const [createLife] = useCreateLifeMutation();
+
+    const onFinish = async values => {
+        const formData = {
+            ...values,
+        };
+
+        if (values['birthday']) {
+            formData['birthday'] = new Date(values['birthday']).valueOf() + '';
+        }
+
+        if (values['hobbies']) {
+            formData['hobbies'] = values['hobbies'].split('\n');
+        }
+
+        await createLife({ variables: formData });
+
+        navigate('/life');
+    };
+
+    return (
+        <BasicLayout>
+            <Form {...layout} name="nest-messages" onFinish={onFinish} validateMessages={validateMessages}>
+                <Form.Item
+                    name={['firstname']}
+                    label="First Name"
+                    rules={[
+                        {
+                            required: true,
+                        },
+                    ]}
```

Comment:
If you only need the rule to be required, there's a `required` prop you can use for `Form.Item`